src/main/java/org/openstreetmap/atlas/checks/validation/tag/AbbreviatedNameCheck.java

```diff
@@ -35,7 +35,7 @@
             "OSM feature with id {0,number,#}'s name tag (`name` = **{1}**) has an abbreviation. Please update the `name` tag to not use abbreviation.");
     // Splitter to parse name
     private static final Splitter NAME_SPLITTER = Splitter
-            .on(CharMatcher.JAVA_LETTER_OR_DIGIT.negate()).omitEmptyStrings();
+            .on(CharMatcher.javaLetterOrDigit().negate()).omitEmptyStrings();
```

Comment:
Latest guava required this. This method is deprecated, so if anyone has a better alternative I am all ears.

Reply:
So an alternative would be something like inRange('0', '9').or(inRange('a', 'z')).or(inRange('A', 'Z')). It is more explicit and not deprecated.

Reply:
Perfect, let me do that.